app/controllers/cloud_volume_controller.rb

```diff
@@ -65,61 +70,6 @@ def button
     end
   end
 
-  def show
-    @display = params[:display] || "main" unless pagination_or_gtl_request?
-    @showtype = @display
-    @lastaction = "show"
-
-    @volume = @record = identify_record(params[:id])
-    return if record_no_longer_exists?(@volume)
-
-    @gtl_url = "/show"
-    drop_breadcrumb({
-                      :name => _("Cloud Volumes"),
-                      :url  => "/cloud_volume/show_list?page=#{@current_page}&refresh=y"},
-                    true)
-
-    case @display
-    when "main", "summary_only"
-      get_tagdata(@volume)
-      drop_breadcrumb(
-        :name => _("%{name} (Summary)") % {:name => @volume.name.to_s},
-        :url  => "/cloud_volume/show/#{@volume.id}"
-      )
-      @showtype = "main"
-      set_summary_pdf_data if @display == 'summary_only'
-    when "cloud_volume_snapshots"
-      title = ui_lookup(:tables => 'cloud_volume_snapshots')
-      kls   = CloudVolumeSnapshot
-      drop_breadcrumb(
-        :name => _("%{name} (All %{children})") % {:name => @volume.name, :children => title},
-        :url  => "/cloud_volume/show/#{@volume.id}?display=cloud_volume_snapshots"
-      )
-      @view, @pages = get_view(kls, :parent => @volume, :association => :cloud_volume_snapshots)
```

Comment:
I see this call to `get_view` has an extra `:association => :cloud_volume_snapshots` argument. This is not done the same when using the generic method. Did you consider that?